frontend/lint/eslint-rules/no-top-level-find-by-text.js

```diff
@@ -0,0 +1,43 @@
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+function rule(context) {
+  return {
+    Identifier(node) {
+      // at the start of analyzing a code path
+      if (isFindByText(node) && hasParentTestBlock(node)) {
+        context.report({
+          node,
+          message:
+            "You should scope your findByText calls to a container using .within()",
```

Comment:
This is not entirely true.
You can scope `.findByText` by simply chaining it to a previous selector.

```js
cy.get("some-parent-element").findByText("string-inside-that-parent");
```